character-controller.js

```diff
@@ -296,46 +351,83 @@ class PlayerBase extends THREE.Object3D {
   }
   getCrouchFactor() {
     return 1 - 0.4 * this.actionInterpolants.crouch.getNormalized();
-    /* let factor = 1;
-    factor *= 1 - 0.4 * this.actionInterpolants.crouch.getNormalized();
-    return factor; */
   }
-  wear(app, {
-    loadoutIndex = -1,
-  } = {}) {
-    const _getNextLoadoutIndex = () => {
-      let loadoutIndex = -1;
-      const usedIndexes = Array(8).fill(false);
-      for (const action of this.getActionsState()) {
-        if (action.type === 'wear') {
-          usedIndexes[action.loadoutIndex] = true;
+  updatePhysics(timestamp, timeDiff) {
+    if (this.avatar) {
+      const timeDiffS = timeDiff / 1000;
+      this.characterPhysics.update(timestamp, timeDiffS);
+    }
+  }
+  handleWearUpdate(app, wear, loadoutIndex = -1, isAppUpdate = false, isPlayerUpdate = true) {

```

Comment:
The arguments to this function are a mess.

When it's this many it should probably be a dict.

Reply:
Also this seems very internal to it can be made private (`#`) to ensure no external callers.